Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -1939,6 +1944,74 @@ def get_machine_details_command(client: Client, args: dict):
             outputs=outputs)
 
 
+def query_malop_management_command(client: Client, args: dict):
+    malop_guid = args.get('malopGuid')
+    json_body = {
+        "search": {
+            "malop": {
+                "guid": f'{malop_guid}'
+            }
+        },
+        "pagination": {
+            "offset": 0
+        },
+        "range": {
+            "from": 0,
+            "to": 9999999999999
+        }
+    }
+    response = client.cybereason_api_call('POST', '/rest/mmng/v2/malops', json_body=json_body)
+    if dict_safe_get(response, ['data', 'data']) == []:
+        raise DemistoException(f"Could not find details for the provided MalopGuid {malop_guid}")
+    else:
+        for single_malop in response["data"]["data"]:
+            guid = single_malop.get("guid", "")
+            creation_time = single_malop.get("creationTime", "")
+            malop_last_update_time = single_malop.get("lastUpdateTime", "")
+            management_status = single_malop.get("investigationStatus", "")
+            involved_hashes = single_malop.get("rootCauseElementHashes", [])
+            if single_malop["isEdr"]:
+                link = SERVER + '/#/malop/' + guid
+            else:
+                link = SERVER + '/#/detection-malop/' + guid
+
+            outputs = []
```

Comment:
please move the `outputs = []` outside the for loop

Reply:
Yes, I have updated the code.